app/src/styles.sass

```diff
@@ -142,18 +138,28 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
 @include angular-material-typography($itero-mat-typography)
 
 
-.sidemenu
+
+.simplemenu, .infomenu
   border-left:1px solid #704f6a
   border-top: 1px solid #704f6a
   border-bottom: 1px solid #704f6a
   background-color: #d5b6d0
   float: right
+  margin-left: 3px
+  box-shadow: -5px 0px 5px 0 rgba(255, 204, 244, 0.7)
+    
+.simplemenu
+  display: none
   height: 100%
   margin-right: -16px
-  margin-left: 3px
-  box-shadow: -2px 0px 2px 0 rgba(255, 204, 244, 0.9)
+.infomenu
+  width: 200px
```

Comment:
Try to avoid using such absolute value in pixels. Here you could use percents. Moreover, I think it could be made wider.